packages/polar/src/builtin-tasks/clean.ts

```diff
@@ -22,15 +24,21 @@ export default function (): void {
       { contractName }: TaskCleanArg,
       env: PolarRuntimeEnvironment
     ) => {
+      const contractNameNew = contractName.toString().replace(/-/g, '_');
```

Comment:
.replace function doesn't replace every `-`, It only replaces the first one, please update this ti replaceAll

Reply:
To make the method replace() replace all occurrences of the pattern you have to enable the global flag on the regular expression:

Append g after at the end of regular expression literal: /search/g
Or when using a regular expression constructor, add 'g' to the second argument: new RegExp('search', 'g')
Let’s replace all occurrences of ' ' with '-':

const searchRegExp = /\s/g;
const replaceWith = '-';
const result = 'duck duck go'.replace(searchRegExp, replaceWith);
result; // => 'duck-duck-go'
The regular expression literal /\s/g (note the g global flag) matches the space ' '.

'duck duck go'.replace(/\s/g, '-') replaces all matches of /\s/g with '-', which results in 'duck-duck-go'.